generate.fsx

```diff
@@ -0,0 +1,118 @@
+open System
```

Comment:
Should this be checked in? It seems like a tool that you are using for the migration but it won't be useful after that.

Reply:
It should indeed be removed. I was planning to wait for this, but as I don't really need it anymore, I'll remove it now.

Reply:
Ok!